src/avatar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cn from 'classnames';
+import img from './missing.png';
+
+import styles from './styles.css';
+
+export default ({ src = '', inline = '', defaultImage = img }) => {
+  const image = src || defaultImage;
```

Comment:
do we need variables for these?
maybe it would be nicer to inline these into the image tag and then return the image tag directly 
```
export default ({ src = '', inline = '', defaultImage = img }) => <img ...
```